src/components/TransactionInputPage.js

```diff
@@ -10,48 +11,37 @@ import {
 
 export default function TransactionInputPage() {
   const [selected, setSelected] = useState(false)
-  const [value, setValue] = useState('')
   const [transactions, setTransactions] = useState([])
-
   useEffect(() => {
     getTransactionEntries().then(setTransactions)
   }, [])
-
   const sum = transactions.reduce(function (acc, transaction) {
     return acc + transaction.value * (transaction.type === 'spending' ? -1 : 1)
   }, 0.0)
-
   return (
     <>
       <ToggleSwitch selected={selected} toggleSelected={handleToggle} />
-      <TransactionFormInput
-        value={value}
-        setValue={setValue}
-        onSave={onSaveAddTransactionEntry}
-      />
+      <TransactionFormInput onSave={onSaveAddTransactionEntry} />
```

Comment:
for better reading I would suggest to rename the onSave function onSaveAddTransactionEntry to handleAddTransactionEntry